### PR TITLE
Styles/feed view

### PR DESCRIPTION
**Pull request by jphorner**
### Features
- Added containers for page components to enable consistent orientation
- Changed main font for app
- Added background color to sentiment options
- Added hover states to sentiment options
- Adjusted width of component displays
- Added borders and spacing to JSX elements

### Refactors 
- Removed form container as the other CSS rules accomplished the same goal
- Added comments within CSS files to explain styling decisions

### Issues and Technical Debt
- 

### Other Comments, Concerns, or Questions
- 

### Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature work
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -1,10 +1,27 @@
+html {
+  background-color: #4B658B;
+  height: 100%;
+  font-family: 'Comfortaa';
+}
+
```
Comment (jphorner) [inline src/components/App/App.css:6]: Here, HTML rules are applied to ensure there is no white space when components are re-sized.

File: src/components/Card/Card.css [hunk 2]
```diff
@@ -2,3 +2,32 @@
...
   width: auto;
 }
+
+.card-container {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
```
Comment (jphorner) [inline src/components/Card/Card.css:12]: The card container has no direct styling and only serves to orient the contents
of each card. As explained in Feed.css, applying flex-row to the container
prevents any context issues caused by the native centering in React.

File: src/components/Feed/Feed.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.articles-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
```
Comment (jphorner) [inline src/components/Feed/Feed.css:6]: The article container is set to flex-row to keep the feed centered on the page.
This means that flex-column can be applied to the articles display while
keeping the row-centered orientation.

React natively has some center orientation applied, but it can cause issues when
orienting individual elements. Applying flex-center to a container overrides the
native centering in the cascade and makes it easier to troubleshoot styling
issues.

File: src/components/Form/Form.css [hunk 4]
```diff
@@ -18,21 +19,41 @@ article {
...
+.sentiments-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  justify-content: space-around;
+  height:120px;
+  width: 50%;
+  background-color: #42C7E4;
+}
+
```
Comment (jphorner) [inline src/components/Form/Form.css:38]: Here, .sentiments-container holds the buttons while .sentiment-selection serves
  to display the light blue stripe and keep .sentiments-container centered.

  To make this work, .sentiment-selection and .sentiments-container both have
  flex applied independently. This is to avoid spacing the buttons via margins
  or padding, which would cause them to shift position when the hover state
  is activated.

  To summarize, .sentiment-selection has a width of 100% to hold the blue stripe and
  to center the container holding the buttons, and .sentiments-container is set to
  50% width + space-around to match the other page elements and enable non-
  intrusive hover states.

**Review by nzambonivergara — CHANGES_REQUESTED** [review]
Great job, I really like the styling of our application! I left one minor comment, and I am attaching a screenshot of how the buttons look when I view our application for mobile devices using the dev tools. I am not entirely sure what is happening there, or if it is just in my device, but maybe something we can look into together?
Screen Shot 2021-11-01 at 10 57 52 AM

File: src/components/App/App.css [hunk 5]
```diff
@@ -1,10 +1,29 @@
+html {
```
[the remark this answers is not in the document]
Comment (jphorner) [reply src/components/App/App.css:1]: Thanks for the review! Yes, I can move the html rule into index.css, and I'll apply some media queries for screen re-sizing. (I set the width of the elements to 50% a bit arbitrarily. I'll be sure to use defined mobile screen sizes from now on.)

I'll do that today and hopefully we'll get it resolved for a merge soon.

File: src/components/App/App.css [hunk 6]
```diff
@@ -1,10 +1,27 @@
...
+}
+
+.header-text-2 {
+  color: #9eeeff;
+}
+
 .App-header {
-  background-color: #282c34;
-  min-height: 20vh;
+  /* background-color: #282c34; */
```
[the remark this answers is not in the document]
Comment (jphorner) [reply src/components/App/App.css:21]: Whoops - will do! My mistake.

File: src/components/Form/Form.tsx [hunk 7]
```diff
@@ -13,23 +13,23 @@ const Form = ({ changeUserSentiment } : FormProps) : JSX.Element => {
...
-          <p>WellNews uses your mood to curate a list of articles with your mental health in mind.</p>
-          <p>A negative mood will prioritize positive articles. <br/>A positive mood will result in a mix of positive, neutral, and negative news.</p>
-          <p>If you are unsure, select a neutral mood for a mix of articles that lean towards the positive.</p>
-          <p>You can change your mood preference at any time.</p>
-        </article>
-      </section>
-    </div>
+        <div className="sentiment-selection">
+          <div className="sentiments-container">
+            <button className="happy-btn" name="1" onClick={assignSentiment}></button>
```
Comment (nzambonivergara) [inline src/components/Form/Form.tsx:21]: The accessibility audits are flagging the fact that our buttons are empty. The solution WAVE provides: "Place text content within the <button> element or give the <input> element a value attribute." Just making a note for us to discuss how to address this.

Comment (jphorner) [reply src/components/Form/Form.tsx:21]: Good catch. I think assigning a value of 'happy', etc. would be the best option to avoid interference with the image itself. If that sounds good enough, I can add 'happy', 'neutral', and 'sad' values during my refactor this afternoon.

File: src/components/Feed/Feed.css (hunk 3 above)
Comment (najinl) [reply src/components/Feed/Feed.css:6]: I think we might need to add a CSS property for the article-image class if we are going to go the route of phone application, as when I set my dev tools to iPhone 6/7/8 or Galaxy S5, the article images start to overflow the articles-container.

File: src/components/Form/Form.tsx (hunk 7 above)
Comment (jphorner) [reply src/components/Form/Form.tsx:21]: (GH deleted my response for some reason? Let me try again.)

Good catch - I think assigning values of 'happy', 'neutral', and 'sad' would be the best option since we don't want text on the buttons. I can add those during my refactor this afternoon if that sounds good to you!

File: src/components/Feed/Feed.css (hunk 3 above)
Comment (najinl) [reply src/components/Feed/Feed.css:6]: I think we might need to add a CSS property for the article-image class if we are going to go the route of phone application, as when I set my dev tools to iPhone 6/7/8 or Galaxy S5, the article images start to overflow the articles-container.
Screen Shot 2021-11-01 at 11 29 04 AM

File: src/components/Form/Form.tsx (hunk 7 above)
Comment (najinl) [reply src/components/Form/Form.tsx:21]: This is something that should be on my docket to update. I'm about to push my changes with router and will update this when I'm in there!

File: src/components/Feed/Feed.css (hunk 3 above)
Comment (jphorner) [reply src/components/Feed/Feed.css:6]: Got it! I'll start adding media queries to resolve this during my refactor. I noticed that happening this morning, too.

File: src/components/Form/Form.tsx (hunk 7 above)
Comment (jphorner) [reply src/components/Form/Form.tsx:21]: Perfect! Thank you, Nadia. I'll just focus on the CSS changes for now to avoid merge conflicts. :)

File: src/components/App/App.css (hunk 5 above)
[the remark this answers is not in the document]
Comment (jphorner) [reply src/components/App/App.css:1]: Good call. I'll be sure to move it.

**Comment by najinl** [comment]
Thank you for make the app BEAUTIFUL! Appreciate you.

**Comment by jphorner** [comment]
**Update:**

- Added media queries to cover all screen sizes
- Moved background styling from App.css to index.css
- Removed commented-out rules

**To be changed later on:**

- Resizing still feels a bit jittery
- Inconsistent margin ratios; may need to use pixel units
- More consistent image sizing


All that said, there should be no major issues when viewing from multiple screen sizes. Let me know what you think!

**Review by nzambonivergara — APPROVED** [review]

**Review by najinl — APPROVED** [review]
